proto/store/setting.proto

```diff
@@ -33,6 +33,22 @@ message WorkspaceProfileSetting {
 
   // The duration for refresh token.
   google.protobuf.Duration refresh_token_duration = 6;
+
+  // We support four types of SMTP authentication: INFO, WARNING, and ALERT.
+  enum AlertLevel {
+    ALERTLEVEL_INFO = 0;
+    ALERTLEVEL_WARNING = 1;
+    ALERTLEVEL_ALERT = 2;
```

Comment:
ALERT -> ERROR.

Using alert is ambiguous, because it translates to "Alert level is alert"